### DISCUSSION TITLE
Remote end hung up -- cloning repo

### DISCUSSION THREAD

**Discussion by joelrb** [Chia-Network/chia-blockchain#4650] [Support]
There's a problem with the repo? Been trying this for like ten times.

jb@jb-desktop:~$ git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules
Cloning into 'chia-blockchain'...
remote: Enumerating objects: 43426, done.
remote: Counting objects: 100% (1752/1752), done.
remote: Compressing objects: 100% (773/773), done.
fatal: the remote end hung up unexpectedly6 MiB | 17.00 KiB/s  
fatal: early EOF
fatal: index-pack failed

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
